openquakeplatform_ipt/static/ipt/js/exposure.js

```diff
@@ -468,13 +484,64 @@ $('.ex_gid #convertBtn').click(function() {
     validateAndDisplayNRML(nrml, 'ex', ex_obj);
 });
 
+function table_file_mgmt(evt)
```

Comment:
Please see if it's not too complicated to remove the duplication of this function